components/org.wso2.carbon.identity.oauth.dcr/src/main/java/org/wso2/carbon/identity/oauth/dcr/util/ErrorCodes.java

```diff
@@ -22,5 +22,7 @@
  */
 public enum ErrorCodes {
     META_DATA_VALIDATION_FAILED,
-    BAD_REQUEST;
+    BAD_REQUEST,
+    FORBIDDEN;
+
```

Comment:
Redundant new line

Reply:
These code changes were reverted recently. I'll fix this with the new PR.